### ISSUE TITLE
What should be our "computational effort" metric

### ISSUE THREAD

**Issue by j-faria** [j-faria/EPRVlnZ#3]
The requirement is a
    integer roughly proportional to how much computational effort was used 
    (e.g., number of iterations, number of model evaluations, CPU-seconds).

This is "effectively a label that allows one method to provide multiple estimates with more or less computational effort."

Maybe number of likelihood evaluations is the easiest metric

**Comment by eggplantbren** [comment]
Yeah, number of DNest iterations is probably best. This will equal the number of likelihood evaluations if you structure things in the standard way so that `log_likelihood()` computes the log likelihood from scratch.
